python/emitter.js

```diff
@@ -201,12 +210,11 @@
             return addCommand(python`${element}.send_keys(${keys})`)
         },
         scrollIntoView(element, align) {
-            console.log('scroll into view Need to be implemented')
-            if (openPerformed) return addCommand(python`eyes_driver.execute_script("arguments[0].scrollIntoView(arguments[1])", ${findElementFunc(element)}, ${align});`)
+			let alignTemp = (align) ? align : false
+            if (openPerformed) return addCommand(python`eyes_driver.execute_script("arguments[0].scrollIntoView(arguments[1])", ${findElementFunc(element)}, ${alignTemp});`)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
            if (openPerformed) {return addCommand(python`eyes_driver.execute_script("arguments[0].scrollIntoView(arguments[1])", ${findElementFunc(element)}, ${alignTemp});`)}
```